lib/inputstreamhelper.py

```diff
@@ -256,6 +256,14 @@ def _inputstream_version(self):
         addon = xbmcaddon.Addon(self.inputstream_addon)
         return addon.getAddonInfo('version')
 
+    @staticmethod
+    def _get_lib_version(lib):
+        if lib:
+            with open(lib, 'rb') as f:
+                x = re.search('[0-9]+\.[0-9]+\.[0-9]+\.[0-9]+', str(f.read())) # pylint: disable=anomalous-backslash-in-string
```

Comment:
When you define a regex pattern in Python, use a `r` before the quotation marks. Pylint won't complain anymore.
For instance:
```python
x = re.search(r'[0-9]+\.[0-9]+\.[0-9]+\.[0-9]+', str(f.read()))
```

Reply:
done.